pages/order-page.pug

```diff
@@ -0,0 +1,37 @@
+extend ../pages/layout
+block page
+	include ../pages/partials/header-info
+block unique content
+	+showNav('Trang chủ / ', '', 'Danh sách đơn hàng')
+	div.order
+		+tittle('ĐƠN HÀNG', 'assets/image/titleleft-dark.png')
+		div.order-detail
+			table.order-detail__table
+				tr
+					td(colspan='7')
+						h4 Danh sách đơn hàng
+						-const stateValue = ['Trạng thái tất cả(16)', 'Đã hủy(10)', 'Chưa duyệt(0)', 'Đã duyệt(0)', 'Đã thanh toán - chờ mua hàng(0)', 'Đã mua hàng(0)', 'Hàng đã về - chờ giao hàng(0)', 'Đã kết thúc(6)']
+						each state in stateValue
+							span=state
+				tr
+					-const headValue = ['STT', 'MÃ HÓA ĐƠN/ NGÀY MUA HÀNG', 'KHO NHẬN HÀNG', 'SỐ SP', 'TỔNG TIỀN', 'TÌNH TRẠNG', 'THAO TÁC']
+					each head in headValue
+						th=head
+				-const tableRowNumber=['01', '02', '03', '04', '05', '06']
+				each tdValue in tableRowNumber
```

Comment:
Ở đây chị có thể sử dụng vòng lặp for, thay vì khai báo 1 mảng tableRowNumber như này

Reply:
okie e